commons-lang/src/main/java/org/apache/commons/lang3/ObjectUtils.java

```diff
@@ -622,7 +622,7 @@ public static <T> T defaultIfNull(final T object, final T defaultValue) {
      */
     @Deprecated
     public static boolean equals(final Object object1, final Object object2) {
-        return object1 == object2;
+        return Objects.equals(object1, object2);
```

Comment:
This changes code needs more review

Reply:
@Acesif please no code logic changes just code formatting changes.